### PR TITLE
✅ Add keybr.com

### PR DESCRIPTION
**Pull request by PiggyPlex**
ignore the dodgy coding, i did it in like an hour :/
(ref #2996)

_In order to property create presences, please take a look at the [documentation](https://docs.premid.app/dev/presence). Take a look at the guidelines to see what is acceptable._

**Screenshots**
Please upload screenshots to github showing what the presence looks like in action. _Do not use third-party image hosting websites._
image
image
image
image

> After your pull request has been made, you should have a screenshot and the code.

### CODE REVIEW

**Comment by Suhalb** [comment]
ty man

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]
Codacy issues

**Comment by NMW03** [comment]
add `closes #2995` to your comment. Github will automatically close that issue when this PR is merged

File: websites/K/keybr.com/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "keybr"
+    ],
+    "category": "games",
+    "settings": [
+        {
+            "id": "changedetails",
+            "title": "Change top line",
+            "icon": "fas fa-trophy",
+            "value": "Current Key: {currentkey}",
+            "placeholder": "Current Key: {currentkey}"
```
Comment (Bas950) [inline websites/K/keybr.com/dist/metadata.json:28]: use %var%

File: websites/K/keybr.com/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        },
+        {
+            "id": "changestate",
+            "title": "Change second line",
+            "icon": "fas fa-info",
+            "value": "WPM: {speed.wpm} | Score: {score}",
+            "placeholder": "WPM: {speed.wpm} | Score: {score}"
```
Comment (Bas950) [inline websites/K/keybr.com/dist/metadata.json:35]: same here

File: websites/K/keybr.com/presence.ts [hunk 3]
```diff
@@ -0,0 +1,165 @@
+const presence: Presence = new Presence({
+  clientId: '812689086435164251'
+});
+
+const startTime: number = new Date().getTime();
```
Comment (Bas950) [inline websites/K/keybr.com/presence.ts:5]: merge consts

File: websites/K/keybr.com/presence.ts [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+    const settings: {
+      [ key: string ]: string
+    } = {
+      details: await presence.getSetting('changedetails'),
+      state: await presence.getSetting('changestate')
+    };
+    const _log = console.log;
+    console.log = () => {};
+    console.log(settings);
+    console.log = _log;
```
Comment (Bas950) [inline websites/K/keybr.com/presence.ts:22]: !?!?!?!?

File: websites/K/keybr.com/presence.ts [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+      if (g && typeof g === 'object') {
+        errors = g.value;
+        errorIncrease = g.increase;
+      };
+      g = _helper2('span.Practice-score');
+      let score, scoreIncrease;
+      if (g && typeof g === 'object') {
+        score = g.value;
+        scoreIncrease = g.increase;
+      };
```
Comment (Bas950) [inline websites/K/keybr.com/presence.ts:85]: sorry what

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/K/keybr.com/presence.ts (hunk 4 above)
Comment (PiggyPlex) [reply websites/K/keybr.com/presence.ts:22]: Settings does not update (for me) without the log line. Callum said that the whole thing is broken for him

File: websites/K/keybr.com/presence.ts (hunk 5 above)
Comment (PiggyPlex) [reply websites/K/keybr.com/presence.ts:85]: Some very readable code. WIll merge consts later

**Comment by ririxidev** [comment]
No response from pull request author. Feel free to do new pr when you will fix issues.

**Pull request closed without merging**